jsk_apc2016_common/node_scripts/polygon_array_to_collision_object.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+
+import time
+
+import tf
+from std_msgs.msg import String
+from geometry_msgs.msg import PointStamped
+from geometry_msgs.msg import PoseStamped
+from geometry_msgs.msg import Quaternion
+from geometry_msgs.msg import TransformStamped
+from geometry_msgs.msg import Vector3
+from jsk_recognition_msgs.msg import PolygonArray
+import moveit_commander
```

Comment:
`moveit_python` includes all `moveit_commander` functions, and it will be merged to `moveit` repo.
The important thing is `moveit_python` has less bug and well-maintained.